### ISSUE TITLE
Is this still being maintained?

### ISSUE THREAD

**Issue by neberej** [lukaszbanasiak/yahoo-finance#124]
I see lot of PRs not being merged.

I have been working on a similar module that combines data from yahoo api, yql api and google finance:

https://github.com/neberej/pinance

I would be happy to raise a PR here.

**Comment by lukaszbanasiak** [comment]
@neberej yes its still maintained, I need to find time to review all changes.

> I would be happy to raise a PR here.

You'r welcome
